python-frontend/typeshed_serializer/tests/conftest.py

```diff
@@ -47,6 +47,11 @@ def typeshed_custom_stubs():
     assert len(build_result.errors) == 0
     return build_result
 
+@pytest.fixture(scope="session")
+def microsoft_stubs():
+    build_result, _ = MicrosoftStubsSerializer().get_build_result()
+    assert len(build_result.errors) == 323
```

Comment:
Does it even make sense to have an assert of this if it's not relevant?
I guess some kind of assert on the build result content to ensure it contains what we want makes sense, but why this one in particular?

Reply:
Yes this was mainly to acknowledge that there are errors we do not have to fix. I would add some assertions that's a good point